system/Session/Session.php

```diff
@@ -36,59 +38,59 @@ class Session implements SessionInterface
     protected $driver;
 
     /**
-     * The storage driver to use: files, database, redis, memcached
+     * The session storage driver to use.
      *
      * @var string
      */
-    protected $sessionDriverName;
+    protected $handler = FileHandler::class;
```

Comment:
@MGatner Isn't renaming protected property is a breaking change?

Reply:
I believe the he kept the original, just moved it lower and deprecated it. I'm on mobile so it's hard to see but I think I saw that during my review.